doc/user/content/ingest-data/postgres/self-hosted.md

```diff
@@ -261,7 +261,40 @@ scenarios, we recommend separating your workloads into multiple clusters for
 [resource isolation](https://materialize.com/docs/sql/create-cluster/#resource-isolation).
 {{< /note >}}
 
-{{% postgres-direct/create-a-cluster %}}
+![Image of the Create New Cluster flow](/images/console/console-create-new/postgresql/create-new-cluster-flow.png "Create New Cluster flow")
```

Comment:
https://preview.materialize.com/materialize/30516/ingest-data/postgres/self-hosted/#c-ingest-data-in-materialize